assets/home/home.scss

```diff
@@ -70,6 +72,10 @@ $break-medium: 783px; // adminbar goes big
 		}
 	}
 
+	.postbox {
+		height: 100%;
```

Comment:
I think I added this instruction for the quick-links section (selector `&__quick-links` on this file) just because of this exact same scenario.

Not essential, but I think it might be good to remove the `height: 100%` from there given this applies that style to every box on the page. :)

Reply:
Ah! Good point! Removed here: https://github.com/Automattic/sensei/pull/5986/commits/9291eb2f97140dd870e10ccf44a8464b9b2f812d